### DISCUSSION TITLE
"wikiimage_sub_blueprint" & "wikimage_sub_lighting" command.

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [FakeFishGames/Barotrauma#14484] [Ideas and suggestions]
so I was reminded of the fact that the game generates those little blueprint pictures for the status monitor.

Since wikiimage_sub is a thing, and we already have that 'blueprint' (find edges) openGL shader: 
it'd be a neat feature to be able to get a rendered output like wikiimage but in that blueprint style.

Now I know it's pretty easy to just run the output png from 'wikiimage' through the filter, but I figured I'd post the suggestion since alot of the baro sub community really has no idea how to do all of that and it would be really convenient for them.

- **Edit:**
a command to high-res screenshot the submarine in editor with lighting enabled would also be useful, as this is not at all obtainable via either previous method. ( previously the only way is to manually capture mosaic image. )

**Comment by jigardi** [comment]
Have my upvote!
